lib/tidewave/tools/list_project_files.rb

```diff
@@ -12,15 +12,15 @@ class Tidewave::Tools::ListProjectFiles < Tidewave::Tools::Base
     By default, when no arguments are passed, it returns all files in the project that
     are not ignored by .gitignore.
 
-    Optionally, a glob_pattern can be passed to filter this list. When a pattern is passed,
-    the gitignore check will be skipped.
+    Optionally, a glob_pattern can be passed to filter this list.
   DESC
 
   arguments do
     optional(:glob_pattern).maybe(:string).description("Optional: a glob pattern to filter the listed files. If a pattern is passed, the .gitignore check will be skipped.")
+    optional(:include_ignored).maybe(:boolean).description("Optional: whether to include files that are ignored by .gitignore. Defaults to false. WARNING: Use with targeted glob patterns to avoid listing excessive files from dependencies or build directories.")
```

Comment:
```suggestion
    optional(:include_ignored).filled(:boolean).description("Optional: whether to include files that are ignored by .gitignore. Defaults to false. WARNING: Use with targeted glob patterns to avoid listing excessive files from dependencies or build directories.")
```